# Each value is between -100 to 100.  0 is a special case that cuts power to the power
int8 left_front
int8 left_middle
int8 left_back
int8 right_front
int8 right_middle
int8 right_back
int8 arm_middle
int8 arm_bottom